### PR TITLE
Solve bug: loading all networks in DB from Openstack.

### PR DESCRIPTION
**Pull request by hmunfru**
#### Reviewers
@jframos @jesuspg

#### Description
https://jira.tid.es/browse/CLAUDIA-4455

#### Test
tested locally

### CODE REVIEW

**Comment by jesuspg** [comment]
LGTM

File: core/src/main/java/com/telefonica/euro_iaas/paasmanager/claudia/impl/ClaudiaClientOpenStackImpl.java [hunk 1]
```diff
@@ -342,25 +343,44 @@ private void addNetworkToTierInstance(ClaudiaData claudiaData, TierInstance tier
...
             }
 
         }
     }
 
+    private NetworkInstance getFirstNetwork(List<NetworkInstance> lNetworkInstance, String vdc, String region)
+        throws InvalidEntityException {
+        NetworkInstance netInstance = null;
+        try {
+            netInstance = networkInstanceManager.load(lNetworkInstance.get(0)
```
Comment (flopezag) [inline core/src/main/java/com/telefonica/euro_iaas/paasmanager/claudia/impl/ClaudiaClientOpenStackImpl.java]: The only comment that I have is related to the fact that we take the networkInstance number 0. It scares me if it is not the correct network to use. Can we assume that it is always the public network?

**Comment by hmunfru** [comment]
it works in that way:
- the user asks for a network
- if the user does not ask, we check if the user has networks deployed, so that, we pick the first one
- if the user does not have any one, we create one for him

**Comment by flopezag** [comment]
LGTM

**Pull request merged**